### ISSUE TITLE
SignaturePad component

### ISSUE THREAD

**Issue by stsrki** [Megabit/Blazorise#4338]
Based on https://github.com/szimek/signature_pad

**Comment by paulsterling** [comment]
Some inspiration here https://github.com/MarvinKlein1508/SignaturePad Also MIT licensed.

I just switched from the (now archived) https://www.nuget.org/packages/Mobsites.Blazor.SignaturePad/ to this and all working as expected so far.

**Comment by stsrki** [comment]
Thanks, I've already seen it. Our component will probably have a similar API once we start working on it.

**Comment by mtbayley** [comment]
I was just about to suggest this component, This would be a great addition!

**Comment by stsrki** [comment]
> @stsrki,
> 
> Possible to add Disabled and/or ReadOnly properties to the SignaturePad component?
> 
> This could be done by calling:
> 
> sigpad.off(); sigpad.on();

Done!

**Comment by sks-ttaylor** [comment]
@stsrki 

I have a local copy of Blazorise.SignaturePad.

Found that the Clear method is not working.

JSSignaturePadModule Clear was not passing elementId

Changed::

I added the following to SignaturePad.razor.cs:
 protected override bool ShouldAutoGenerateId => true;

Modified JSSignaturePadModule  to add parameter "string elementId" to all that reference elementId in the signaturepad.js code.

Example:
export function clear(element, elementId) {
  const instance = _instances[elementId];  <<<< without ShouldAutoGenerateId = true, elementId is null

  if (instance && instance.sigpad) {
    instance.sigpad.clear();
  }
}
